docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,73 @@
+[id='assembly_building-custom-dashboard-widgets']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Building custom dashboard widgets
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: building-custom-dashboard-widgets
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a business analyst or business rules developer, use the *Page Editor* tool to design and define your page and specify what information to be displayed on it. You can decide which components (core, navigational, or reporting) are a part of your page. You can also customize the component properties of a page according to your requirements. However, you should define a data provider before creating a page.
+
```

Comment:
You can decide which components are a part of your page. You can also customize the component properties of a page according to your requirements. Optionally, before creating a page, you can define a data set to feed the reporting a page might have.

Reply:
Sentence rephrased as suggested.